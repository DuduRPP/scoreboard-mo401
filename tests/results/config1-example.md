| Instruction     |   Issue |   Read |   Execute |   Write |
|:----------------|--------:|-------:|----------:|--------:|
| fld f1, x1      |       1 |      2 |         3 |       4 |
| fld f5, x1      |       2 |      3 |         4 |       5 |
| fdiv f2, f4, f5 |       3 |      6 |        16 |      17 |
