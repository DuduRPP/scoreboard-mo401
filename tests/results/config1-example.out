| Instruction     |   Issue |   Read |   Execute |   Write |
|:----------------|--------:|-------:|----------:|--------:|
| fld f1, x1      |       1 |      2 |         3 |       4 |
| fld f5, x1      |       2 |      3 |         4 |       5 |
| fdiv f2, f4, f5 |       3 |      5 |        15 |      16 |
